src/devices/device.ts

```diff
@@ -126,6 +135,22 @@ export class Device extends EventEmitter {
         return this._isVirtualPort;
     }
 
+    /**
+     * @readonly
+     * @property {string[]} events List of availlable events (input modes).
+     */
+    public get events () {
```

Comment:
Something about the naming of `events` and `writeModes` doesn't sit well with me, the naming is quite inconsistent between them despite them doing similar things. Not sure what the ideal naming is here though.

Perhaps it would be better to name it `autoParsedEvents`? Since these are events determined by auto parsing, and not the human friendly names emitted by the devices themselves.

Reply:
I think user friendly mode could be defined the same way auto parsed modes are defined, this way this could return the available events in both cases.

I also think my naming is awful, maybe `inputs` and `outputs` would be better.